scripts/app.js

```diff
@@ -0,0 +1,356 @@
+const { promisify } = require("util");
+const exec = promisify(require("child_process").exec);
+const fs = require("fs");
+const dotenv = require("dotenv");
+const result = dotenv.config();
+
+if (result.error) {
+  if(result.error.errno === -2){
+    console.log("The .env file does not exist. You cannot stage a build without this file.")
+    return
+  }
+}
+
+const { parsed: envs } = result;
+
+if(process.env.DB_NAME === undefined || process.env.DB_NAME === ""){
```

Comment:
In order to make a nice user experience, can we wrap these in a function and keep the validation state within the function, if the call fails validation, format the error like:

".env file is found but does not contain the following required fields: [DB_NAME, COL_NAME, etc]"